src/Components/Registration/registrationForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Button, Form, Spinner } from "react-bootstrap";
+import "./registrationForm.css";
+
+import { RegistationRequest } from "../../Redux/createAction/createAction";
+import { useHistory } from "react-router-dom";
+
+const Registration = () => {
+  const [username, setusername] = useState("");
+  const [password, setpassword] = useState("");
+  const [option, setoptions] = useState("Admin");
+  const history = useHistory();
+
+  const dispatch = useDispatch();
+  const state = useSelector((state) => {
+    return state;
+  });
+
+  const regisstatus = state.Regis_status;
+  const handlesubmit = () => {
+    let formData = {
+      username: username,
+      password: password,
+      option: option,
+    };
+    dispatch(RegistationRequest(formData));
+    setusername("");
+    setpassword("");
+    if (regisstatus.isLoading === false && regisstatus.isRegistered === false) {
```

Comment:
what is this condition ?

Reply:
this is  redundant code .I have removed it